.size-limit.cjs

```diff
@@ -6,9 +6,9 @@ module.exports = [
   },
   {
     name: "Waku default setup",
```

Comment:
Could you repalce this with "Waku Simple Light Node"

And import the all the APIs that are used in the "getting started" example. I think that'd be a better indicator.